cmd/root.go

```diff
@@ -214,46 +214,6 @@ func deriveNameAndAbsolutePathFromPath(path string) (string, string) {
 	return pathParts[len(pathParts)-1], absPath
 }
 
-// deriveImage returns the same image name which will be used.
```

Comment:
Removing these 40 lines (along with the others like them) is a great sign we're headed in the right direction